src/util/proxyDevices.tsx

```diff
@@ -0,0 +1,152 @@
+  index: number,
+  connectionType: ConnectionType,
+  formik: InstanceAndProfileFormikProps,
+  headingTitle: string,
+) => {
+  const deviceParts = device[connectionType]?.split(":") || [];
+  const deviceType = deviceParts.length > 0 ? deviceParts[0] : "tcp";
+  const deviceAddress = deviceParts.length > 1 ? deviceParts[1] : "";
+  const devicePort = deviceParts.length > 2 ? deviceParts[2] : "";
+
+  customRows.push(
+    getConfigurationRowBase({
+      className: "no-border-top inherited-with-form p-heading--6",
+      configuration: headingTitle,
+      inherited: <></>,
+      override: "",
+    }),
+  );
+
+  customRows.push(
+    getConfigurationRowBase({
+      className: "no-border-top inherited-with-form",
+      configuration: (
+        <Label forId={`devices.${index}.${connectionType}`}>Type</Label>
+      ),
+      inherited: (
+        <Select
+          id={`devices.${index}.${connectionType}`}
+          onChange={(e) => {
+            ensureEditMode(formik);
+            const newType = e.target.value;
+            void formik.setFieldValue(
+              `devices.${index}.${connectionType}`,
+              `${newType}:${deviceAddress}:${devicePort}`,
+            );
+
+            if (device.nat === "true") {
+              const connectParts = device.connect?.split(":") || [];
+              const connectAddress =
+                connectParts.length > 1 ? connectParts[1] : "";
+              const connectPort =
+                connectParts.length > 2 ? connectParts[2] : "";
+
+              void formik.setFieldValue(
+                `devices.${index}.connect`,
+                `${newType}:${connectAddress}:${connectPort}`,
+              );
+            }
+          }}
+          value={deviceType}
+          options={proxyAddressTypeOptions}
+          className="u-no-margin--bottom"
+          disabled={connectionType === "connect" && device.nat === "true"}
+          title={
+            device.nat
+              ? "This is determined by the listen type when nat mode is enabled"
+              : undefined
+          }
+        />
+      ),
+      override: "",
+    }),
+  );
+
+  customRows.push(
+    getConfigurationRowBase({
+      className: "no-border-top inherited-with-form",
+      configuration:
+        deviceType === "unix" ? (
+          <Label forId={`devices.${index}.unixsocket`}>*Socket path</Label>
+        ) : (
+          <Label forId={`devices.${index}.address`}>*Address</Label>
+        ),
+      inherited:
+        deviceType === "unix" ? (
+          <Input
+            id={`devices.${index}.unixsocket`}
+            onChange={(e) => {
+              ensureEditMode(formik);
+              const socketPath = e.target.value;
+              void formik.setFieldValue(
+                `devices.${index}.${connectionType}`,
+                `unix:${socketPath}`,
+              );
+            }}
+            value={deviceAddress}
+            placeholder="/<socket_path_on_instance>"
```

Comment:
```suggestion
            placeholder="/<socket_path>"
```
Sorry for going back and forth here. I just realized in listen the socket path on instance is wrong, and it should be socket path on host. The easiest solution is to make the copy generic.